controller/array_action/messages.py

```diff
@@ -14,6 +14,8 @@
 
 SPACE_EFFICIENCY_NOT_SUPPORTED_MESSAGE = "space efficiency is not supported : {0} "
 
+FLASHCOPY_2_NOT_SUPPORTED_MESSAGE = "flashcopy 2.0 is enabled but not supported with object : {0} "
```

Comment:
shouldn't it be `with storage : {0}` (and pass the management address instead)?

Reply:
not necessarily, in #502 we change the check for the source as well, and fcmap in source would raise it too, so it's object-based

Reply:
I see. it's hard to keep track when both PRs are open